main.py

```diff
@@ -2,41 +2,41 @@
 import time
 from datetime import timedelta
 from car import Car
+#from cars import Cars
 import geopandas as gpd
 import osmnx as ox
 import networkx as nx
 from pyproj import CRS
 import pandas as pd
 from request_generator import RequestGenerator
 from request import Request
+from PyQt5 import uic, QtWidgets"""to bedzie do GUI"""
 
 
+#update wszystkich pozycji i statystyk samochodów naraz
+def update_all(car_list):

```

Comment:
Generalnie ja chciałam zrobić coś takiego, żeby wrzucić tę listę do jakiejś klasy. I wtedy tę funkcję update_all() zamknąć w klasie. Bo to jest trochę niedobre, żeby ona była w pliku main. To powinna być chyba tylko funkcja main(). Niemniej jednak...